services/src/main/java/org/jboss/windup/web/services/rest/WindupEndpoint.java

```diff
@@ -69,4 +69,8 @@
     @GET
     @Path("coreVersion")
     String getCoreVersion();
+
+    @GET
+    @Path("reloadRules")
+    void reloadRules();
```

Comment:
IMHO this should go to `RuleEndpoint`. 

Btw we have `reloadConfiguration();` in `ConfigurationEndpoint` which does basically the same thing, so I don't think we need this